drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -956,6 +956,8 @@ void nrf_802154_receive_failed(nrf_802154_rx_error_t error, uint32_t id)
 		break;
 	}
 
+	LOG_INF("Rx failed, error = %d", error);
```

Comment:
In a very noisy environment this log may printed very often. Is it really necessary to print this? Can it be a debug level log instead?

Reply:
The very noisy environment is where we would take the most benefit from the informational log here. If we change to debug log level we end up with many additional logs which will obfuscate the whole picture.

Reply:
This is code is used by everyone. This logging could be under some `#if`. If you need so deep logging then you can enable this option on your own.
Possibly:
```c
#if defined(CONFIG_IEEE802154_NRF5_LOG_RX_FAILURES)
   LOG_INF("Rx failed, error = %d", error);
#endif
```
and `config IEEE802154_NRF5_LOG_RX_FAILURES` option turned off by default in nearby Kconfig.nrf5

Reply:
We can set log level for this specific module/file so you can still disable logs if you are not interested in. There is already Kconfig for that: `CONFIG_IEEE802154_DRIVER_LOG_LEVEL` so we don't need to add extra one.
What I find most interest among all logs in this module is this what I want to add here but combining it with others with the debug log level has not much sense to me as it will not be much help in that shape.

Reply:
@lmaciejonczyk  yes, you do. By doing so you will have the feature you need, other users of shared shim will not be affected.